extensions/src/main/java/org/apache/falcon/extensions/jdbc/ExtensionMetaStore.java

```diff
@@ -154,7 +157,7 @@ public void storeExtensionJob(String jobName, String extensionName, List<String>
         EntityManager entityManager = getEntityManager();
         try {
             beginTransaction(entityManager);
-            entityManager.persist(extensionJobsBean);
+            entityManager.merge(extensionJobsBean);
```

Comment:
But, the store is updated only once, irrespective of how many processes and feeds are are in job, isn't it?